umm-spec-lib/src/cmr/umm_spec/migration/version/service.clj

```diff
@@ -173,6 +174,198 @@
     "WEB SERVICES"
     service-type))
 
+(defn create-main-url-for-1_3
+  "When migrating from UMM-S v1.2 to UMM-S v1.3 verison 1.2 RelatedURLs needs to be
+   converted to the new URL element.  Take the first DistributionURL element from
+   RelatedURLs and convert it to the new URL type."
+  [s]
+  (let [url (->> (:RelatedURLs s)
+                 (filter #(= (:URLContentType %) "DistributionURL"))
+                 first)]
+    (when url
+      (-> url
+          (assoc :URLValue (:URL url))
+          (dissoc :URL)))))
+
+(defn create-main-related-urls-for-1_2
+  "Migrating from UMM-S v1.3 to UMM-S v1.2 RelatedURLs."
+  [s]
+  (let [url (:URL s)]
+    (when url
+      [(-> url
+           (assoc :URL (:URLValue url))
+           (dissoc :URLValue))])))
+
+(defn create-online-resource
+  "Create an online resource structure for service organization staring in version 1.3.
+   Since Description is required in 1.3, if it doesn't exist then add the words Not provided
+   as the description."
+  [service-org]
+  (let [url (->> (:ContactInformation service-org)
+                 (:RelatedUrls)
+                 (filter #(= (:URLContentType %) "DataCenterURL"))
```

Comment:
You can use `some` here instead of `filter-first` (see other comment).

Reply:
See the above comment.  The some function is returning a boolean true instead of a map from a vector of maps.